.baseimagedigest

```diff
@@ -0,0 +1,2 @@
+sha256:4ba4d3e2da3a7edb8a8e79fd38e95702f445aa4d5a66a1b1667260b5f1405acc
+e7f572acdd67188c711ece321f47b9cb88afdd2decacc5133887b99d8a2605a8  -
```

Comment:
@vkrizan what's the `-` at the end?

Reply:
stdin

https://linux.die.net/man/1/sha256sum

Reply:
https://github.com/RedHatInsights/floorist/pull/43